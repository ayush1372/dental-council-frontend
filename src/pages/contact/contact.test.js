import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';

import { Contact } from './index';

test('Contact component snapshot', () => {
  const { container } = render(
    <Router>
      <Contact />
    </Router>
  );
  expect(container).toMatchSnapshot();
});
