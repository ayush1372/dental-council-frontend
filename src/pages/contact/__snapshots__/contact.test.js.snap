// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component snapshot 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
  >
    <div
      class="MuiBox-root css-1wcaknn"
    >
      <div
        class="MuiBox-root css-1yado2t"
      >
        <h2
          class="MuiTypography-root MuiTypography-h2 css-wf8t3v-MuiTypography-root"
        >
          Get In Touch With Us For More Information
        </h2>
      </div>
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-5 css-urj5pv-MuiGrid-root"
        >
          <div
            class="MuiBox-root css-1wcee1m"
          >
            <div
              class="MuiBox-root css-7t14gn"
            >
              <div
                class="MuiBox-root css-1fhgjcy"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="LocationOnOutlinedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 2C8.13 2 5 5.13 5 9c0 5.25 7 13 7 13s7-7.75 7-13c0-3.87-3.13-7-7-7zM7 9c0-2.76 2.24-5 5-5s5 2.24 5 5c0 2.88-2.88 7.19-5 9.88C9.92 16.21 7 11.85 7 9z"
                  />
                  <circle
                    cx="12"
                    cy="9"
                    r="2.5"
                  />
                </svg>
              </div>
              <div
                class="MuiBox-root css-c7pnka"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                >
                  ADDRESS
                </p>
                <div
                  class="MuiBox-root css-0"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                  >
                    9th Floor, Tower-l, Jeevan Bharati Building, Connaught Place, New Delhi - 110001
                  </p>
                </div>
              </div>
            </div>
            <div
              class="MuiBox-root css-1rwr66l"
            >
              <div
                class="MuiBox-root css-1fhgjcy"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="CallOutlinedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M6.54 5c.06.89.21 1.76.45 2.59l-1.2 1.2c-.41-1.2-.67-2.47-.76-3.79h1.51m9.86 12.02c.85.24 1.72.39 2.6.45v1.49c-1.32-.09-2.59-.35-3.8-.75l1.2-1.19M7.5 3H4c-.55 0-1 .45-1 1 0 9.39 7.61 17 17 17 .55 0 1-.45 1-1v-3.49c0-.55-.45-1-1-1-1.24 0-2.45-.2-3.57-.57-.1-.04-.21-.05-.31-.05-.26 0-.51.1-.71.29l-2.2 2.2c-2.83-1.45-5.15-3.76-6.59-6.59l2.2-2.2c.28-.28.36-.67.25-1.02C8.7 6.45 8.5 5.25 8.5 4c0-.55-.45-1-1-1z"
                  />
                </svg>
              </div>
              <div
                class="MuiBox-root css-0"
              >
                <div
                  class="MuiBox-root css-c7pnka"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                  >
                    CONTACT US
                  </p>
                  <div
                    class="MuiBox-root css-0"
                  >
                    <div
                      class="MuiBox-root css-0"
                    >
                      <p
                        class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                      >
                        Toll-Free Number : 1800-11-4477 
                      </p>
                    </div>
                    <div
                      class="MuiBox-root css-0"
                    >
                      <p
                        class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                      >
                        E-mail: abdm@nha.gov.in 
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiBox-root css-1euowjw"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
              >
                Note:- Please do not file your grievance on “CONTACT US”, instead kindly address it on our grievance portal by visiting
                 
                <a
                  class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1em2h7l-MuiTypography-root-MuiLink-root"
                >
                  https://grievance.abdm.gov.in
                </a>
                 and register your grievances there.
              </p>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-7 css-1a4ufwv-MuiGrid-root"
        >
          <div
            class="MuiBox-root css-f38rbf"
          >
            <iframe
              frameborder="0"
              height="539"
              id="gmap_canvas"
              marginheight="0"
              marginwidth="0"
              scrolling="no"
              src="https://maps.google.com/maps?q=9th%20Floor%2C%20Tower-L%2C%20Jeevan%20Bharati%20Building%2C%20Connaught%20Place%2C%20New%20Delhi%20-%20110001&t=&z=13&ie=UTF8&iwloc=&output=embed"
              title="Google Maps"
              width="100%"
            />
          </div>
        </div>
      </div>
      <div
        class="MuiBox-root css-68zlnx"
      >
        <div
          class="MuiBox-root css-0"
        >
          <div
            class="MuiTabs-root MuiTabs-vertical css-1g94vql-MuiTabs-root"
          >
            <div
              class="MuiTabs-scroller MuiTabs-fixed css-jpln7h-MuiTabs-scroller"
              style="overflow: hidden; margin-right: 0px;"
            >
              <div
                aria-orientation="vertical"
                class="MuiTabs-flexContainer MuiTabs-flexContainerVertical css-lfwcke-MuiTabs-flexContainer"
                role="tablist"
              >
                <button
                  aria-selected="true"
                  class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary Mui-selected css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                  role="tab"
                  tabindex="0"
                  type="button"
                >
                  General FAQs
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
                <button
                  aria-selected="false"
                  class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                  role="tab"
                  tabindex="-1"
                  type="button"
                >
                  Other FAQs
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
                <button
                  aria-selected="false"
                  class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                  role="tab"
                  tabindex="-1"
                  type="button"
                >
                  Miscellaneous
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </div>
              <span
                class="MuiTabs-indicator css-10d9dml-MuiTabs-indicator"
                style="top: 0px; height: 0px;"
              />
            </div>
          </div>
        </div>
        <div
          class="MuiBox-root css-1g35dz1"
        >
          <div
            class="MuiBox-root css-0"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation1 MuiAccordion-root MuiAccordion-gutters css-t4qmgb-MuiPaper-root-MuiAccordion-root"
            >
              <div
                aria-controls="panel1bh-content"
                aria-expanded="false"
                class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-sh22l5-MuiButtonBase-root-MuiAccordionSummary-root"
                id="panel1bh-header"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters css-o4b71y-MuiAccordionSummary-content"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-vkc4v8-MuiTypography-root"
                  >
                    What is ABHA number ?
                  </p>
                </div>
                <div
                  class="MuiAccordionSummary-expandIconWrapper css-yw020d-MuiAccordionSummary-expandIconWrapper"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="ExpandMoreIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
                    />
                  </svg>
                </div>
              </div>
              <div
                class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-bz4dnt-MuiCollapse-root"
                style="min-height: 0px;"
              >
                <div
                  class="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
                >
                  <div
                    class="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
                  >
                    <div
                      aria-labelledby="panel1bh-header"
                      class="MuiAccordion-region"
                      id="panel1bh-content"
                      role="region"
                    >
                      <div
                        class="MuiAccordionDetails-root css-15v22id-MuiAccordionDetails-root"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-7aqg5k-MuiTypography-root"
                        >
                          ABHA number is a 14 digit number that will uniquely identify you as a participant in India’s digital healthcare ecosystem. ABHA number will establish a strong and trustable identity for you that will be accepted by healthcare providers and payers across the country
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation1 MuiAccordion-root MuiAccordion-gutters css-t4qmgb-MuiPaper-root-MuiAccordion-root"
            >
              <div
                aria-controls="panel2bh-content"
                aria-expanded="false"
                class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-sh22l5-MuiButtonBase-root-MuiAccordionSummary-root"
                id="panel2bh-header"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters css-o4b71y-MuiAccordionSummary-content"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-vkc4v8-MuiTypography-root"
                  >
                    What is ABDM ?
                  </p>
                </div>
                <div
                  class="MuiAccordionSummary-expandIconWrapper css-yw020d-MuiAccordionSummary-expandIconWrapper"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="ExpandMoreIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
                    />
                  </svg>
                </div>
              </div>
              <div
                class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-bz4dnt-MuiCollapse-root"
                style="min-height: 0px;"
              >
                <div
                  class="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
                >
                  <div
                    class="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
                  >
                    <div
                      aria-labelledby="panel2bh-header"
                      class="MuiAccordion-region"
                      id="panel2bh-content"
                      role="region"
                    >
                      <div
                        class="MuiAccordionDetails-root css-15v22id-MuiAccordionDetails-root"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-7aqg5k-MuiTypography-root"
                        >
                          ABHA (Ayushman Bharat Health Account) Address is a unique identifier (self declared username) that enables you to share and access your health records digitally. Your ABHA address may look like ‘yourname@consent manager’.For instance, xyz@abdm is a ABHA address with ABDM Consent Manager that will facilitate health data exchange for you with appropriate consent on the ABDM network
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation1 MuiAccordion-root MuiAccordion-gutters css-t4qmgb-MuiPaper-root-MuiAccordion-root"
            >
              <div
                aria-controls="panel3bh-content"
                aria-expanded="false"
                class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-sh22l5-MuiButtonBase-root-MuiAccordionSummary-root"
                id="panel3bh-header"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters css-o4b71y-MuiAccordionSummary-content"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-vkc4v8-MuiTypography-root"
                  >
                    Linking ABHA number with ABHA address
                  </p>
                </div>
                <div
                  class="MuiAccordionSummary-expandIconWrapper css-yw020d-MuiAccordionSummary-expandIconWrapper"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="ExpandMoreIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
                    />
                  </svg>
                </div>
              </div>
              <div
                class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-bz4dnt-MuiCollapse-root"
                style="min-height: 0px;"
              >
                <div
                  class="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
                >
                  <div
                    class="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
                  >
                    <div
                      aria-labelledby="panel3bh-header"
                      class="MuiAccordion-region"
                      id="panel3bh-content"
                      role="region"
                    >
                      <div
                        class="MuiAccordionDetails-root css-15v22id-MuiAccordionDetails-root"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-7aqg5k-MuiTypography-root"
                        >
                          You can use your ABHA number to seamlessly sign up for a ABHA address, and ensure that the health records created for you are shared only with you. To enable health data sharing, it is recommended that you create ABDM ABHA address and link it with your ABHA number .
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation1 MuiAccordion-root MuiAccordion-gutters css-t4qmgb-MuiPaper-root-MuiAccordion-root"
            >
              <div
                aria-controls="panel4bh-content"
                aria-expanded="false"
                class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-sh22l5-MuiButtonBase-root-MuiAccordionSummary-root"
                id="panel4bh-header"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters css-o4b71y-MuiAccordionSummary-content"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-vkc4v8-MuiTypography-root"
                  >
                    Voluntary Opt-out ?
                  </p>
                </div>
                <div
                  class="MuiAccordionSummary-expandIconWrapper css-yw020d-MuiAccordionSummary-expandIconWrapper"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="ExpandMoreIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
                    />
                  </svg>
                </div>
              </div>
              <div
                class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-bz4dnt-MuiCollapse-root"
                style="min-height: 0px;"
              >
                <div
                  class="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
                >
                  <div
                    class="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
                  >
                    <div
                      aria-labelledby="panel4bh-header"
                      class="MuiAccordion-region"
                      id="panel4bh-content"
                      role="region"
                    >
                      <div
                        class="MuiAccordionDetails-root css-15v22id-MuiAccordionDetails-root"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-7aqg5k-MuiTypography-root"
                        >
                          it is Voluntary that you create ABDM ABHA address and link it with your ABHA number .
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation1 MuiAccordion-root MuiAccordion-gutters css-t4qmgb-MuiPaper-root-MuiAccordion-root"
            >
              <div
                aria-controls="panel5bh-content"
                aria-expanded="false"
                class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-sh22l5-MuiButtonBase-root-MuiAccordionSummary-root"
                id="panel5bh-header"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters css-o4b71y-MuiAccordionSummary-content"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-vkc4v8-MuiTypography-root"
                  >
                    What is HIE-CM ?
                  </p>
                </div>
                <div
                  class="MuiAccordionSummary-expandIconWrapper css-yw020d-MuiAccordionSummary-expandIconWrapper"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="ExpandMoreIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
                    />
                  </svg>
                </div>
              </div>
              <div
                class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-bz4dnt-MuiCollapse-root"
                style="min-height: 0px;"
              >
                <div
                  class="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
                >
                  <div
                    class="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
                  >
                    <div
                      aria-labelledby="panel5bh-header"
                      class="MuiAccordion-region"
                      id="panel5bh-content"
                      role="region"
                    >
                      <div
                        class="MuiAccordionDetails-root css-15v22id-MuiAccordionDetails-root"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-7aqg5k-MuiTypography-root"
                        >
                          To enable health data sharing, it is recommended that you create ABDM ABHA address and link it with your ABHA number .
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation1 MuiAccordion-root MuiAccordion-gutters css-t4qmgb-MuiPaper-root-MuiAccordion-root"
            >
              <div
                aria-controls="panel6bh-content"
                aria-expanded="false"
                class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-sh22l5-MuiButtonBase-root-MuiAccordionSummary-root"
                id="panel6bh-header"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters css-o4b71y-MuiAccordionSummary-content"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-vkc4v8-MuiTypography-root"
                  >
                    How do i get ABHA Number ?
                  </p>
                </div>
                <div
                  class="MuiAccordionSummary-expandIconWrapper css-yw020d-MuiAccordionSummary-expandIconWrapper"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="ExpandMoreIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
                    />
                  </svg>
                </div>
              </div>
              <div
                class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-bz4dnt-MuiCollapse-root"
                style="min-height: 0px;"
              >
                <div
                  class="MuiCollapse-wrapper MuiCollapse-vertical css-smkl36-MuiCollapse-wrapper"
                >
                  <div
                    class="MuiCollapse-wrapperInner MuiCollapse-vertical css-9l5vo-MuiCollapse-wrapperInner"
                  >
                    <div
                      aria-labelledby="panel6bh-header"
                      class="MuiAccordion-region"
                      id="panel6bh-content"
                      role="region"
                    >
                      <div
                        class="MuiAccordionDetails-root css-15v22id-MuiAccordionDetails-root"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-7aqg5k-MuiTypography-root"
                        >
                          it is ABHA Number that you create ABDM ABHA address and link it with your ABHA number .
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
